rsocket-core/src/main/java/io/rsocket/core/RequestOperator.java

```diff
@@ -115,6 +114,7 @@ public void request(long n) {
       if (firstLoop) {
         firstLoop = false;
         try {
+          this.s.request(Long.MAX_VALUE);
```

Comment:
An implementation comment on here would be helpful I think as it is not obvious. Some condensed summary of https://github.com/rsocket/rsocket-java/issues/959#issuecomment-782025770 also mentioning that the queue in `UnicastProcessor` is of size 1.